### PR TITLE
Cleanout Elgg doc and internal config

### PR DESCRIPTION
**Pull request by nschonni**
Since GCConnex isn't setup as a fork to contribute back, may as well cleanup the Elgg internal docs since those aren't modified by this project

### CODE REVIEW

**Comment by nschonni** [comment]
@Phanoix even in you 1.12.6 PR, you have disabled that travis script https://github.com/gctools-outilsgc/gcconnex/pull/1187/commits/0775fbc0398452f04d19d17dd0a81f464de7955f

**Comment by Phanoix** [comment]
I meant the rest of  .travis.yml, the commit message checking being removed should be fine for how we're using it right now.

File: .travis.yml [hunk 1]
```diff
@@ -6,65 +6,7 @@ php:
...
-       - pip install --user "sphinx-intl"
-       - export PATH=$PATH:$HOME/.local/bin
-      script:
-       - sphinx-build -b html -nW docs docs/_build/html
-       - sphinx-build -b latex -nW docs docs/_build/latex
-       - sphinx-intl build --locale-dir=docs/locale/
-       - sphinx-build -b html -D language=es -n docs docs/_build/html
-
-    # End to end tests
-    - php: 5.6
```
Comment (nschonni) [inline .travis.yml]: @Phanoix was this the one you wanted to restore?

Comment (Phanoix) [inline .travis.yml]: that's it, the one line for the comment formatting can go but everything else is being used.

**Comment by nschonni** [comment]
Rebased and added back the package.json to keep the JS tests in the matrix

**Comment by ToferC** [comment]
Approve review of code pull.

**Comment by Phanoix** [comment]
I'll take a closer look at this before merging on Tuesday or over the weekend.

**Pull request merged**